packs/feats/assisting-shot.json

```
@@ -28,7 +28,27 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "assisting-shot",
+                "toggleable": true
+            },
+            {
+                "key": "Note",
+                "outcome": [
+                    "criticalSuccess",
+                    "success"
+                ],
+                "predicate": [
+                    "assisting-shot"
+                ],
+                "selector": "ranged-attack-roll",
+                "text": "PF2E.SpecificRule.Fighter.AssistingShot.Note",
```

```suggestion
                "text": "{item|system.description.value}",
```
The description isn't so long that it requires us to trim it down for localization.